resources/helm/v2.0/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -147,7 +147,7 @@
     defaultConfig:
       #
       # TCP connection timeout between Envoy & the application, and between Envoys.
-      connectTimeout: 10s
+      #connectTimeout: 10s
```

Comment:
did this cause issues?

Reply:
Apparently, it's no longer in the schema and galley was barfing when it tried deserializing it.  I don't think it affected validation, but it did emit an error in the istiod logs.  (Also, it was a bit of an iterative process trying to figure out how to test this, so this might have been a problem I encountered along the way that may have been a red herring.)